microsoft/onefuzz
# Update az cli 2.47

## Summary of the Pull Request
Updating AZ CLI to 2.47 to patch [a bicep module bug](https://github.com/Azure/azure-cli/issues/25710) along with several other supporting python dependencies. 

## PR Checklist
* [x] Applies to work item: (ADO) # 45421

## Info on Pull Request
Updated:
* src/cli/requirements.txt
* src/deployment/requirements.txt
* src/utils/add-corpus-storage-accounts/requirements.txt
* src/utils/check-pr/requirements.txt

## Validation Steps Performed
Running our check-pr pipeline in ADO, the 'Deploy' stage runs successfully but has the error:
 `ERROR: No section: 'bicep'`
